### PR TITLE
Solved the task

### PR DESCRIPTION
**Pull request by Cerne13**

### CODE REVIEW

**Review by TPolina — CHANGES_REQUESTED** [review]

File: cinema/views.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    if request.method == "DELETE":
+        movie.delete()
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+    return Response("Method not allowed", status=status.HTTP_405_METHOD_NOT_ALLOWED)
+
+
+
```
[the remark this answers is not in the document]
Comment (Cerne13) [reply cinema/views.py:54]: Sorry for these stupid mistakes. Was very sleepy when coded. Will fix asap.

File: cinema/serializers.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField()
+    duration = serializers.IntegerField(required=True)
+
+    def create(self, validated_data):
+        return Movie.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("info", instance.title)
```
[the remark this answers is not in the document]
Comment (Cerne13) [reply cinema/serializers.py:16]: Strange. It's "title" in the local repo. Maybe I haven't pushed something...

File: cinema/serializers.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=63)
+    description = serializers.CharField(max_length=255)
```
Comment (TPolina) [inline cinema/serializers.py:9]: if you want to make a description not required field you need to specify this in serializer. If you want it to be required you need to fix model

**Review by TPolina — CHANGES_REQUESTED** [review]

File: cinema/serializers.py (hunk 3 above)
Comment (Cerne13) [reply cinema/serializers.py:9]: Oops, sorry...

**Review by MasterpieceElbow — APPROVED** [review]
GJ!

**Pull request closed without merging**